spec/workers/atmosphere/vm_template_monitoring_worker_spec.rb

```diff
@@ -13,8 +13,10 @@
     it { should be_processed_in :monitoring }
   end
 
-  context 'updating cloud site images' do
-    let(:cyfronet_folsom) { create(:compute_site, site_id: 'cyfronet-folsom',config: '{"provider": "openstack", "openstack_auth_url": "http://10.100.0.2:5000/v2.0/tokens", "openstack_api_key": "key", "openstack_username": "user"}') }
+  context 'updating tenant images' do
+    let(:cyfronet_folsom) { create(:tenant, tenant_id: 'cyfronet-folsom', \
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.